noxfile.py

```diff
@@ -49,15 +49,15 @@ def run_integration(session):
     session.env[
         "LD_LIBRARY_PATH"
     ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
-    session.install("-e", ".[dev]")
+    session.install("-e", ".[all]")
     if sys.platform == "linux":
         session.install("scikits.odes")
     session.run("python", "run-tests.py", "--integration")
 
 
 @nox.session(name="doctests", reuse_venv=True)
 def run_doctests(session):
-    session.install("-e", ".[docs]")
+    session.install("-e", ".[all,docs]")
```

Comment:
Should this also be just `docs`?

Reply:
These are also requiring other dependencies as well, i remember `pybtex`, `tqdm` and etc. so I ended up installing `[all]` too.

Reply:
> Should this also be just `docs`?

`[dev,docs]` could work here.

Reply:
I tried that but [tests](https://github.com/pybamm-team/PyBaMM/actions/runs/5447374473/jobs/9909365784#step:15:24) were failing so currently installing `[all]`.